### PR TITLE
Y24-266   LCM Triomics pipeline configuration and integration test for EMSeq branch

### PR DESCRIPTION
**Pull request by yoldas**
Closes #

#### Changes proposed in this pull request

- Submission, request type and library type for the EMSeq branch

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: config/default_records/product_catalogues/013_lcm_triomics_catalogue.wip.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+LCM Triomics WGS and EMSeq:
```
Comment (andrewsparkes) [inline config/default_records/product_catalogues/013_lcm_triomics_catalogue.wip.yml:2]: rename to LCM Triomics (see limber PR comment)

File: config/default_records/submission_templates/014_lcm_triomics_submission_templates.wip.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Submission templates for LCM Triomics WGS and EMSeq
+---
+# LCM Triomics EMSeq submission template
+Limber-Htp - LCM Triomics EMSeq:
+  submission_class_name: "LinearSubmission"
+  related_records:
+    request_type_keys: ["limber_lcm_triomics_emseq"]
+    product_line_name: LCM Triomics WGS and EMSeq
```
Comment (andrewsparkes) [inline config/default_records/submission_templates/014_lcm_triomics_submission_templates.wip.yml:8]: I would rename these to just LCM Triomics too

**Review by andrewsparkes — CHANGES_REQUESTED** [review]
Just some rename suggestions as for Limber

File: config/default_records/product_catalogues/013_lcm_triomics_catalogue.wip.yml (hunk 1 above)
Comment (yoldas) [reply config/default_records/product_catalogues/013_lcm_triomics_catalogue.wip.yml]: This affects the grouping of Submission Templates in Limber, e.g., dropdown control in Create Submission page or the listing in Create Batch Submission pages. Setting the group name to LCM Triomics makes sense. I have now changed the product catalogue name.

File: config/default_records/submission_templates/014_lcm_triomics_submission_templates.wip.yml (hunk 2 above)
Comment (yoldas) [reply config/default_records/submission_templates/014_lcm_triomics_submission_templates.wip.yml]: I have renamed them both catalogue and product line to just LCM Triomics now.

**Review by andrewsparkes — APPROVED** [review]

**Pull request merged**